types/bramus__pagination-sequence/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for @bramus/pagination-sequence 1.2
+// Project: https://github.com/bramus/js-pagination-sequence#readme
+// Definitions by: Kristján Jökull Sigurðsson <https://github.com/kristjanjokull>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function generate(
+    curPage: number,
+    numPages: number,
+    numPagesAtEdges?: number,
+    numPagesAroundCurrent?: number,
+    glue?: string,
+): Array<number | string>;
+
+interface Options {
+    curPage?: number;
+    numPages?: number;
+    numPagesAtEdges?: number;
+    numPagesAroundCurrent?: number;
+    glue?: string;
+}
+
+declare function generateFromObj(opts: Options): Array<string | number>;
```

Comment:
this one should take optional parameter IMO, as source uses default parameter value, readme:
https://github.com/bramus/js-pagination-sequence/tree/main#generatefromobj
```suggestion
declare function generateFromObj(opts?: Options): Array<string | number>;
```